src/test-suite/generate_llfiles.sh

```diff
@@ -5,20 +5,20 @@
 # file in the top-level directory for more details.
 #
 SRC_WD="PE-benchmarks"
-DEST_FOLDER="PE-benchmarks-llfiles-llvm12"
+DEST_FOLDER="PE-benchmarks-llfiles-llvm14"
 
 mkdir -p ${DEST_FOLDER}
 
 # Update the BUILD to use
-LLVM_BUILD=
+LLVM_BUILD="/usr"
 
-if [ -z ${LLVM_BUILD} ]; then
-	echo "Enter the llvm build path.."
-	exit
-fi
+# if [ -z ${LLVM_BUILD} ]; then
```

Comment:
Pl revert back

Reply:
the LLVM_BUILD path is never used in the script. So, I've removed it. Please check if it needs to be reinstated nonetheless